bridges/GBAtempBridge.php

```diff
@@ -32,45 +32,62 @@ private function buildItem($uri, $title, $author, $timestamp, $thumbnail, $conte
 		return $item;
 	}
 
+	private function strEndsWith($haystack, $needle){
+		// str_ends_with is not available below PHP 8
+		$length = strlen($needle);
+		return $length > 0 ? substr($haystack, -$length) === $needle : true;
+	}
```

Comment:
Just backported str_ends_with. https://github.com/RSS-Bridge/rss-bridge/pull/2318

Reply:
Thanks for adding backports into rss-bridge! Switched to the default `str_ends_with` function 👍